tests/TestCase/Http/ResponseTest.php

```diff
@@ -749,26 +749,6 @@ public function testWithCookieEmpty(): void
         $this->assertEquals($expected, $result);
     }
 
-    /**
-     * Test setting cookies with scalar values
-     */
-    public function testWithCookieScalar(): void
```

Comment:
These tests don't make much sense. They are testing the value stored in the Cookie object which is not the value sent and retrieved via headers. Those are expected to be strings.